lib/assets/SnapshotHelper.swift

```diff
@@ -36,18 +37,31 @@ class Snapshot: NSObject {
         let path = "/tmp/language.txt"
 
         do {
-            locale = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding) as String
-            deviceLanguage = locale.substringToIndex(locale.startIndex.advancedBy(2, limit:locale.endIndex))
-            app.launchArguments += ["-AppleLanguages", "(\(deviceLanguage))", "-AppleLocale", "\"\(locale)\"", "-ui_testing"]
+            deviceLanguage = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding) as String
```

Comment:
Hi @i2amsam,
Your change will be working, but I've made "more" changes because I've also added support for custom locales using `setLocale`. With `setLocale` users would be able to use completely different locale, not related to the selected language. Most users will probably never use this option, but Xcode allows to set such configuration and maybe someone need this.